server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -299,7 +293,6 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 		private readonly rawDeltasProducer: IProducer,
 		private readonly serviceConfiguration: IServiceConfiguration,
 		private sessionMetric: Lumber<LumberEventName.SessionResult> | undefined,
-		private readonly sessionStartMetric: Lumber<LumberEventName.StartSessionResult> | undefined,
```

Comment:
Why remove this metric? I think reliability might use this?

Reply:
Hi @zhangxin511, based on the details above, it is likely not providing an accurate reliability measurement. @pradeepvairamani thought it would be best to remove this. We I discussed that we have better ways of measuring reliability, such as `RunSession` metrics for the individual lambdas.

Reply:
@pradeepvairamani do you have any additional reasons?

Reply:
The main reason is that this metric is just a consolidation of deli start and scribe start and to get this telemetry info, we are also introducing a control op and writing to checkpoint which all makes this quite expensive to just get this telemetry info. We already bubble up the errors during lambda start, but if we want we can add individual lambda start metrics which should be much cleaner.

Reply:
Thank you @kekachmar and @pradeepvairamani . I was just confirming. With this being said, is there a metric we could use to replace for this?
![image](https://github.com/microsoft/FluidFramework/assets/3719246/a1852f49-ef1c-45c7-96e7-6deadb8c7c7b)

Reply:
@zhangxin511 / @pradeepvairamani, do we want to address this further and add individual lambda start metrics?

Reply:
@kekachmar , we can build this metric later .